kobo/apps/audit_log/tests/test_project_history_logs.py

```diff
@@ -1868,22 +1853,22 @@ def test_update_qa_data(self, is_anonymous, expected_username):
         instance, submission = self._add_submission(
             'adminuser' if not is_anonymous else None
         )
+        question_uuid = self.asset.advanced_features['_actionConfigs']['q1']['qual'][0][
+            'uuid'
+        ]
         log_metadata = self._base_project_history_log_test(
-            method=self.client.post,
+            method=self.client.patch,
             url=reverse(
-                'advanced-submission-post',
-                kwargs={'asset_uid': self.asset.uid},
+                'api_v2:submission-supplement',
```

Comment:
Can't we use the helper `self._get_endpoint()` everywhere `api_v2` is hardcoded instead?